python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/sling/asset_defs.py

```diff
@@ -123,11 +123,9 @@ def build_assets_from_sling_stream(
     Args:
         source (SlingConnectionResource): The source SlingConnectionResource to use.
         target (SlingConnectionResource): The target SlingConnectionResource to use.
-        stream (str): The source stream to sync from. This can be a table, a query, or a path.
+        streams (str): The streams to sync from and any configurations specific to the stream. This can be a table, a query, or a path. See the Sling documentation for more information. https://docs.slingdata.io/sling-cli/replication
```

Comment:
link doesnt work, should be https://docs.slingdata.io/sling-cli/run/configuration#replication-config

Reply:
Oh hey, looks like flarco re-did the docs. cool! thanks for calling it out.

And yeah, this needs a better docstring. will update after our call.